post_processing/plot/plot_helpers.jl

```diff
@@ -582,4 +616,92 @@ function generate_paperplots_held_suarez(fig_dir, nc_files; moist)
     end
 
     CairoMakie.save(fig_dir * "/diagnostics.png", fig)
+
+    # sfc_flux_energy, sfc_evaporation, lw_flux_diff, sw_flux_diff are set to
+    # nothing if there's no flux information
+    has_flux = any(
+        map(
+            !isnothing,
+            [sfc_flux_energy, sfc_evaporation, lw_flux_diff, sw_flux_diff],
+        ),
+    )
+
+    if has_flux
+        fig_flux = generate_empty_figure(; resolution = (2048, 2048))
+    end
+
+    if !isnothing(sfc_flux_energy)
+        create_plot!(
+            fig_flux;
+            p_loc = (1, 1),
+            X = lon,
+            Y = lat,
+            Z = sfc_flux_energy[:, :, end],
+            title = "Surface flux of energy: Day $(days[end]);",
+            xlabel = "Longitude",
+            ylabel = "Latitude",
+        )
+    end
+    if !isnothing(sfc_evaporation)
+        create_plot!(
+            fig_flux;
+            p_loc = (1, 2),
+            X = lon,
+            Y = lat,
+            Z = sfc_evaporation[:, :, end],
+            title = "Surface evaporation: Day $(days[end]);",
+            xlabel = "Longitude",
+            ylabel = "Latitude",
+        )
+    end
+    if !isnothing(lw_flux_diff)
+        create_plot!(
+            fig_flux;
+            p_loc = (2, 1),
+            X = lon,
+            Y = lat,
+            # There's a minus because we define diff = upward - downward
+            Z = -lw_flux_diff[:, :, end, end],
+            title = "Net longwave flux at TOA: Day $(days[end]);",
```

Comment:
```suggestion
            title = "Net (down - up) longwave flux at TOA: Day $(days[end]);",
```
(Or something similar to this)